scripts/apps/authoring/authoring/services/AutosaveService.js

```diff
@@ -61,6 +67,36 @@ export class AutosaveService {
         return this.timeouts[id];
     }
 
+    /**
+     * Generate item annotations field
+     *
+     * @param {Object} item
+     */
+    generateAnnotations(item) {
```

Comment:
I don't think it should be in autosave service. imo best would be if it's handled in the editor on change

Reply:
Awesome suggestion, thanks :+1: It also required less code to put it in the editor, and that's where it belongs as well.